app/services/send_email_service/send_notify_email.rb

```diff
@@ -43,5 +43,8 @@ def undeliverable_failure?(error)
     return false unless error.is_a?(Notifications::Client::BadRequestError)
 
     error.message.end_with?("Not a valid email address")
```

Comment:
your change would make this line have no consequences. we'd want this to be `return true if error.message.end_with?("Not a valid email address")`

Reply:
Aye I noticed my dumb mistake almost immediately (pinged you on slack). Been a long couple days.